docs/scripting.md

```diff
@@ -481,11 +481,11 @@ Using previous examples:
     # rest of the code here
 
 
-  robot.hear /midnight train/i, (res)
+  robot.hear /midnight train/i, (msg) ->
```

Comment:
This should remain `res` to match other documentation and that it's a `Response` object